example/build.gradle

```diff
@@ -47,3 +48,22 @@ dependencies {
     androidTestImplementation 'androidx.test.ext:junit:1.2.1'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.6.1'
 }
+
+spotless {
+    format 'misc', {
+        // define the files to apply `misc` to
+        target '*.gradle', '.gitattributes', '.gitignore'
+
+        // define the steps to apply to those files
+        trimTrailingWhitespace()
+        indentWithSpaces(2) // or spaces. Takes an integer argument if you don't like 4
```

Comment:
this comment seems outdated

Reply:
For Java I like 4 but it's not a strong preference nor am I am a JVM expert. I defer to @felipecsl's wisdom on indentation!

Reply:
this is actually for all the non .java files. We're applying the "Google java style" to the actual java files which is 2 spaces